### ISSUE TITLE
Localised urls

### ISSUE THREAD

**Issue by alexauvray** [cibernox/svelte-intl-precompile#54]
How to get localised urls like you set on https://svelte-intl-precompile.com/en/docs/introduction ?

In order to have /en/docs and /es/docs.

**Comment by thousandsofraccoons** [comment]
@alexauvray just create folder `[lang]` and put your routes inside there. Then initialize the library from URL parameter.

```
 export async function load({ params, fetch, url }: any) {
    const lang = params.lang.substring(0, 2)}
```

@cibernox I've another question related to the topic. How can I achieve the following structure:
1.  / - default language
2. /es
3. /de

Thank you

**Comment by cibernox** [comment]
@thousandsofraccoons You can check how I did it in the documentation page: https://github.com/cibernox/svelte-intl-precompile-docs

**tl;dr;** Define the route as `[...lang]`, so you have a nesting level that can take zero or more arguments.

**Comment by cibernox** [comment]
@thousandsofraccoons thanks for answering the question! I'm closing this now
